### ISSUE TITLE
Incorrect token length specification

### ISSUE THREAD

**Issue by mattrussell2** [kesyog/crossword#17]
Love this -- thanks for doing it so I didn't have to!

However, the token length of my NYT cookie did not fit the specification currently required. 

Mine is 170 characters.

To be clear, the count is of the characters in the location [[HERE]] in the following string taken from the header args on the NYT website as the instructions indicate. 

```
NYT-S=[[HERE]]; 
```

Removing the check of token length in `api_client.rs` and instead by default inserting the headers which would have been inserted for a count of 162 fixed the issue for me.

**Comment by kesyog** [comment]
Thanks for the heads up! I don't love the existing mechanism that relies on an inflexible string length heuristic to figure out whether the token is a cookie or header.

IMO we should replace the `--nyt_token` CLI arg with `--nyt_header` and `--nyt_cookie` args to force the user to specify what they're passing in. I should be able to put out a fix for this relatively soon.

**Comment by mattrussell2** [comment]
Sure thing - sounds good to me! Thanks again.

Also, one other option might be to keep the same CLI and just to try both? Then if they both fail then report failure to the user...this would probably require more implementation work though :)

**Comment by kesyog** [comment]
Ended up taking the simpler approach and adding args for both, `--nyt-cookie` and `--nyt-header`. `-t` is mapped back to cookie to make things easier for people.

It seems like one can't extract a token from the header anymore. I'm using a header-based token I extracted years ago so the underlying API still works for now, but I assume most people will be using the cookie-based approach.

Thanks again for flagging this!
